content/github/authenticating-to-github/working-with-ssh-key-passphrases.md

```diff
@@ -18,10 +18,10 @@ With SSH keys, if someone gains access to your computer, they also gain access t
 You can change the passphrase for an existing private key without regenerating the keypair by typing the following command:
 
 ```shell
-$ ssh-keygen -p
+$ ssh-keygen -p -f /home/<em>you</em>/.ssh/id_ed25519
```

Comment:
To avoid the issue of `/home/you/...` on Linux and `/Users/you/...` on Mac, I think we should use `~` to cover both
```suggestion
$ ssh-keygen -p -f ~/.ssh/id_ed25519
```